x-pack/plugins/apm/public/components/app/correlations/ml_failure_correlations.tsx

```diff
@@ -0,0 +1,410 @@
+              <ImpactBar size="m" value={normalizedScore * 100} />
+            </>
+          );
+        },
+      },
+      {
+        width: '116px',
+        field: 'pValue',
+        name: (
+          <>
+            {i18n.translate(
+              'xpack.apm.correlations.failure.correlationsTable.impactLabel',
+              {
+                defaultMessage: 'Impact',
+              }
+            )}
+          </>
+        ),
+        render: getFailureCorrelationImpactLabel,
+      },
+      {
+        field: 'fieldName',
+        name: i18n.translate(
+          'xpack.apm.correlations.failure.correlationsTable.fieldNameLabel',
+          { defaultMessage: 'Field name' }
+        ),
+      },
+      {
+        field: 'key',
+        name: i18n.translate(
+          'xpack.apm.correlations.failure.correlationsTable.fieldValueLabel',
+          { defaultMessage: 'Field value' }
+        ),
+        render: (fieldValue: string) => String(fieldValue).slice(0, 50),
+      },
+      {
+        width: '100px',
+        actions: [
+          {
+            name: i18n.translate(
+              'xpack.apm.correlations.correlationsTable.filterLabel',
+              { defaultMessage: 'Filter' }
+            ),
+            description: i18n.translate(
+              'xpack.apm.correlations.correlationsTable.filterDescription',
+              { defaultMessage: 'Filter by value' }
+            ),
+            icon: 'plusInCircle',
+            type: 'icon',
+            onClick: (term: FailureCorrelationValue) => {
+              push(history, {
+                query: {
+                  kuery: `${term.fieldName}:"${encodeURIComponent(
+                    term.fieldValue
+                  )}"`,
+                },
+              });
+              handleOnFilter();
+              trackApmEvent({ metric: 'correlations_term_include_filter' });
+            },
+          },
+          {
+            name: i18n.translate(
+              'xpack.apm.correlations.correlationsTable.excludeLabel',
+              { defaultMessage: 'Exclude' }
+            ),
+            description: i18n.translate(
+              'xpack.apm.correlations.correlationsTable.excludeDescription',
+              { defaultMessage: 'Filter out value' }
+            ),
+            icon: 'minusInCircle',
+            type: 'icon',
+            onClick: (term: FailureCorrelationValue) => {
+              push(history, {
+                query: {
+                  kuery: `not ${term.fieldName}:"${encodeURIComponent(
+                    term.fieldValue
+                  )}"`,
+                },
+              });
+              handleOnFilter();
+              trackApmEvent({ metric: 'correlations_term_exclude_filter' });
+            },
+          },
+        ],
+        name: i18n.translate(
+          'xpack.apm.correlations.correlationsTable.actionsLabel',
+          { defaultMessage: 'Filter' }
+        ),
+        render: (_: any, term: FailureCorrelationValue) => {
```

Comment:
Does it work making the `_` an `unknown`?

Reply:
Updated here https://github.com/elastic/kibana/pull/108441/commits/155443605ee491b2eb7c2bcb7b3e458eed9591e6